### DISCUSSION TITLE
CSE%40GaTech/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#217] [Announcements]
# CSE%40GaTech/

None

https://opencs.app/CSE%40GaTech/

**Comment by yuhengcai1** [comment]
这项目很奇怪，可以考虑往上放，因为这个项目往年录的很多实力偏弱的（我都能看见非CS强校的211出身），但是出路却能往上走很多，项目强度被自己的奇怪的bar给拖累了，如果不看bar的话这个项目应该是UCLA的MSCS的全面上位替代，而且现在COC不给转PHD了（转的人太多），只有CSE学院可以转PHD了

**Reply by yuhengcai1** [reply]
最怪的是，它录一个中流211的非科班生，然后他给转GT的CS PHD，这个跨度基本上约等于高考直接一波加分加到清北叉院

**Reply by HUST-AI-HYZ** [reply]
我看了下那个dp, 根据timeline估计不是coc的学生, 如果录取的是cee track那还算正常，个人觉得gatech cse的coc应该单独拎出来，放在A或者A+，有的dp mscs录取了结果cse coc挂了

**Comment by zzy347964399** [comment]
补一个今年home unit的pdf：https://cse.gatech.edu/sites/default/files/documents/2024/Handbook-2024-08-06.pdf
